build_config.rb.lock

```diff
@@ -0,0 +1,172 @@
+---
+mruby_version:
+  version: 2.1.0
+  release_no: 20100
+builds:
+  host:
+    https://github.com/iij/mruby-env.git:
+      url: https://github.com/iij/mruby-env.git
+      branch: master
+      commit: 056ae324451ef16a50c7887e117f0ea30921b71b
+      version: 0.0.0
+    https://github.com/iij/mruby-dir.git:
+      url: https://github.com/iij/mruby-dir.git
+      branch: master
+      commit: 89dceefa1250fb1ae868d4cb52498e9e24293cd1
+      version: 0.0.0
+    https://github.com/iij/mruby-digest.git:
+      url: https://github.com/iij/mruby-digest.git
+      branch: master
+      commit: 2bd4627d3430921a4290f7622b3e891979d5cee0
+      version: 0.0.0
+    https://github.com/iij/mruby-process.git:
+      url: https://github.com/iij/mruby-process.git
+      branch: master
+      commit: 95da206a5764f4e80146979b8e35bd7a9afd6850
+      version: 0.0.0
+    https://github.com/mattn/mruby-json.git:
+      url: https://github.com/mattn/mruby-json.git
+      branch: master
+      commit: '054699892b0d681110e7348440cddb70a3fd671e'
+      version: 0.0.0
+    https://github.com/mattn/mruby-onig-regexp.git:
+      url: https://github.com/mattn/mruby-onig-regexp.git
+      branch: master
+      commit: e55923997ba33586b58c11bd3ca2d40c11a8dede
```

Comment:
I have a build error as below. 

```
GIT RESET e55923997ba33586b58c11bd3ca2d40c11a8dede 
fatal: Could not parse object 'e55923997ba33586b58c11bd3ca2d40c11a8dede'.
fatal: Could not parse object 'e55923997ba33586b58c11bd3ca2d40c11a8dede'.
rake aborted!
Command failed with status (128): [git --git-dir /home/vagrant/ngx_mruby/mrub...]
```

Reply:
When I build the project, it's generating `build_config.rb.lock` file (probably MRuby new update). 
So, If this file doesn't need to be committed. Should I add it in .gitignore file?